tests/integration/targets/meraki_mr_radio/tasks/main.yml

```diff
@@ -0,0 +1,146 @@
+# Test code for the Meraki modules
+# Copyright: (c) 2021, Tyler Christiansen (@supertylerc)
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- block:
+  - name: Create network with type wireless
+    meraki_network:
+      auth_key: "{{ auth_key }}"
+      state: present
+      org_name: "{{ test_org_name }}"
+      net_name: IntTestNetworkWireless
+      type: wireless
+      timezone: America/Chicago
+    delegate_to: localhost
+
+  - name: Create RF profile
+    meraki_mr_rf_profile:
+      auth_key: "{{ auth_key }}"
+      org_name: "{{ test_org_name }}"
+      net_name: IntTestNetworkWireless
+      state: present
+      name: Test Profile
+      band_selection_type: ap
+      client_balancing_enabled: True
+      ap_band_settings:
+          mode: dual
+          band_steering_enabled: true
+      five_ghz_settings:
+        max_power: 10
+        min_bitrate: 12
+        min_power: 8
+        rxsop: -65
+        channel_width: 20
+        valid_auto_channels:
+          - 36
+          - 40
+          - 44
+      two_four_ghz_settings:
+        max_power: 10
+        min_bitrate: 12
+        min_power: 8
+        rxsop: -65
+        ax_enabled: false
+        valid_auto_channels:
+          - 1
+    delegate_to: localhost
+    register: create
+
+  - set_fact:
+      profile_id: "{{ create.data.id }}"
+
+  - name: Configure AP radios in check mode
```

Comment:
I am receiving this error. I assume you're not. I'll look into this but if you're aware of why it's happening, please update. It may be because of different Ansible versions. I'm on 2.11.6.

```
TASK [meraki_mr_radio : Configure AP radios in check mode] **************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'module' object has no attribute 'JSONDecodeError'
fatal: [localhost -> localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/Users/kbreit/.ansible/tmp/ansible-tmp-1638238599.554506-961-120512832505302/AnsiballZ_meraki_mr_radio.py\", line 114, in <module>\n    _ansiballz_main()\n  File \"/Users/kbreit/.ansible/tmp/ansible-tmp-1638238599.554506-961-120512832505302/AnsiballZ_meraki_mr_radio.py\", line 106, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/Users/kbreit/.ansible/tmp/ansible-tmp-1638238599.554506-961-120512832505302/AnsiballZ_meraki_mr_radio.py\", line 55, in invoke_module\n    run_name='__main__', alter_sys=True)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py\", line 188, in run_module\n    fname, loader, pkg_name)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py\", line 82, in _run_module_code\n    mod_name, mod_fname, mod_loader, pkg_name)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py\", line 72, in _run_code\n    exec code in run_globals\n  File \"/tmp/ansible_meraki_mr_radio_payload_qF_Dxu/ansible_meraki_mr_radio_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_mr_radio.py\", line 478, in <module>\n  File \"/tmp/ansible_meraki_mr_radio_payload_qF_Dxu/ansible_meraki_mr_radio_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_mr_radio.py\", line 474, in main\n  File \"/tmp/ansible_meraki_mr_radio_payload_qF_Dxu/ansible_meraki_mr_radio_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_mr_radio.py\", line 401, in meraki_run\n  File \"/tmp/ansible_meraki_mr_radio_payload_qF_Dxu/ansible_meraki_mr_radio_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_mr_radio.py\", line 371, in _meraki_run_present\n  File \"/tmp/ansible_meraki_mr_radio_payload_qF_Dxu/ansible_meraki_mr_radio_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_mr_radio.py\", line 360, in meraki_get_radio_settings\n  File \"/tmp/ansible_meraki_mr_radio_payload_qF_Dxu/ansible_meraki_mr_radio_payload.zip/ansible_collections/cisco/meraki/plugins/module_utils/network/meraki/meraki.py\", line 478, in request\nAttributeError: 'module' object has no attribute 'JSONDecodeError'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

Reply:
My testing is done on 2.10.x.  It's possibly happening because the profile is set to a list of valid auto-channels and the override is for something outside of that range, but would need the full error back from the Meraki API to confirm.  Unfortunately, I don't have a test environment to verify.  But I'll push an update to the channels anyway to see if that fixes it.

Reply:
Attempted fix is in bc68b50.

Reply:
Not sure if it's related to your PR. I'm seeing this elsewhere and troubleshooting.